syntax = "proto3";

message User {
  uint32 id = 1;
  string name = 2;
}

message UserRequest { uint32 id = 1; }

message UserResponse {
  bool error = 1;
  string message = 2;
  User user = 3;
}

service UserManager {
  rpc getUser(UserRequest) returns (UserResponse) {}
}